front/commonitiltask.form.php

```diff
@@ -93,6 +93,20 @@
     $task->check($_POST["id"], UPDATE);
     $task->update($_POST);
 
+    Event::log(
+        $task->getField($fk),
+        strtolower($itemtype),
+        4,
+        "tracking",
+        //TRANS: %s is the user login
+        sprintf(__('%s updates a task'), $_SESSION["glpiname"])
+    );
+    $redirect = $itemtype::getFormURLWithID($task->getField($fk));
+    $handled = true;
+} else if (isset($_POST["unplan"])) {
+    $task->check($_POST["id"], UPDATE);
+    $task->unplan($_POST);
+
```

Comment:
Following log message should say "unplan" and not "updates", no? Looks good to me appart of that.